scripts/run-httpd-with-fixtures

```diff
@@ -49,16 +49,17 @@ prepare_workspace() {
   echo "Unpacking fixture $FIXTURE"
   mkdir -p $WORKSPACE
   tar -xf $FIXTURE -C $WORKSPACE
-  if [ "$CI" = true ]; then
-    sudo chown -R 0:0 $WORKSPACE
-  fi
 }
 
 run_docker() {
   echo "Starting docker at container $CONTAINER_NAME at $REV"
   echo "  RAD_HOME=$WORKSPACE RAD_PASSPHRASE=$PASSPHRASE radicle-http"
   echo
 
+  if [ "$CI" = true ]; then
+    sudo chown -R 0:0 $WORKSPACE
+  fi
+
```

Comment:
We don't need this anymore, because we'll probably only use the docker version for local development. And CI is now covered by the binary.